app/assets/stylesheets/components/_important-metadata.scss

```diff
@@ -10,6 +10,19 @@
     overflow: auto;
     padding: govuk-spacing(4) govuk-spacing(6);
   }
+
+  @include govuk-media-query($media-type: print) {
+    padding: 0;
```

Comment:
The non print style has some left and right padding, which changes the layout. I'm unclear if that layout change communicates anything important (it seems fairly arbitrary to me). I think it looks fine, but wondering if a designer needs to give it a quick once over?

Reply:
I think it looks fine too - we don't have any guidelines around print styles that I'm aware of. I think this was just indented because it already has padding on the non-print styles.